ci/long_running_tests/workloads/serve_handle.py

```diff
@@ -0,0 +1,88 @@
+# A test that stresses the serve handles. We spin up a backend with a bunch
```

Comment:
this actually doesn't belong to long running test, it's more just a serve benchmark.